server/routes/index.ts

```diff
@@ -1,12 +1,18 @@
-import type { RequestHandler, Router } from 'express'
+import express, { RequestHandler, Router } from 'express'
+import ChoosePrisonerController from './choosePrisonerController'
+import IncomingMovementsService from '../services/incomingMovementsService'
 
 import asyncMiddleware from '../middleware/asyncMiddleware'
 
-export default function routes(router: Router): Router {
+export default function routes(incomingMovementsService: IncomingMovementsService): Router {
+  const router = express.Router()
   const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
 
+  const choosePrisonerController = new ChoosePrisonerController(incomingMovementsService)
+  get('/confirm-arrival/choose-prisoner', choosePrisonerController.view())
```

Comment:
as part of a bit of restructurng we'll probably extract out a `/confirm-arrival` router and restructure this. 
It's fine for now though